<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>floorDesign</class>
 <widget class="QDialog" name="floorDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="fullWidget">
     <item>
      <layout class="QHBoxLayout" name="displayBox">
       <item>
        <layout class="QVBoxLayout" name="editPanelTab">
         <item>
          <layout class="QHBoxLayout" name="editFloorPlanHeader">
           <item>
            <widget class="QLabel" name="panelLabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Panel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="add">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="delete_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="panelTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Panel</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Bottom of Panel</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Top of Panel</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="View3DGLWidget" name="Panel3DViewer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="RIghtBar" stretch="0,0,1,10">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="EditPanel">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Edit Panel</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="DisplayPanelText">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="DisplayLabelName">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="selectedPanelName">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="Coord">
             <property name="text">
              <string>Coordinates</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="CoordTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <row>
            <property name="text">
             <string>Upper Left</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Bottom Left</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Upper Right</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Bottom Right</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>X</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Y</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Z</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="FloorPlan_buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View3DGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">View3DEngine</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
